### ISSUE TITLE
Bringing container up fails when installing APCU

### ISSUE THREAD

**Issue by rowanparker** [dunglas/symfony-docker#139]
Just tried running `docker-compose up -d` and received this message. Changing the `APCU_VERSION` version in the `Dockerfile` makes no difference.

```
#6 18.90 WARNING: Ignoring https://dl-cdn.alpinelinux.org/alpine/v3.13/main: No such file or directory
#6 18.90 WARNING: Ignoring https://dl-cdn.alpinelinux.org/alpine/v3.13/community: No such file or directory
#6 18.90 (1/1) Purging .phpize-deps-configure (20210503.115948)
#6 18.90 OK: 301 MiB in 79 packages
#6 19.18 + pecl install apcu-5.1.19
#6 24.21 No releases available for package "pecl.php.net/apcu"
#6 24.21 install failed
```

**Comment by maxhelias** [comment]
This is a regular issue, we can do nothing, just tried again : https://github.com/dunglas/symfony-docker/issues/134

**Comment by rowanparker** [comment]
@maxhelias 

I know you closed this, but I have investigated it some more. If I comment out the PECL lines, I still get another error when it tries to create the `symfony/skeleton` project.

So I suspected this was a networking problem inside the container. I commented out that part as well and the container actually builds and comes up. If I then go into the container and attempt to run `pecl install apcu-5.1.1` command I get the same error.

`ping pecl.php.net` inside the container and I get `bad address 'pecl.php.net'`
`nslookup pecl.php.net` inside the container and I get a `non-authoritative answer`.

If I add a dns option to docker-compose I can at least ping inside the container
`	dns: 8.8.8.8`

I then try and run `pecl install apcu-5.1.19` in the container and it downloads the file ok. It complains about not finding a `$AUTOCONF` var, but at least I worked out it's DNS now.

So I guess this issue is related to these:
https://github.com/laradock/laradock/issues/2828
https://bugs.php.net/bug.php?id=80693

If I change the `Dockerfile` to specify the older version of Alpine, everything works fine:
`FROM php:${PHP_VERSION}-fpm-alpine3.12`

As per that laradock issue, maybe we should to the same?

**Comment by maxhelias** [comment]
In fact, it seems that this is a problem with docker for mac which is now fixed in docker desktop community 3.1.0 : https://github.com/docker/for-mac/issues/5020  https://github.com/docker/docker.github.io/pull/12096/files https://gitlab.alpinelinux.org/alpine/aports/-/issues/11879

I tried an installation with a higher version and I don't have any problems. Can you confirm this on your side?

**Comment by maxhelias** [comment]
Anyway, it's not related to this project and there is nothing we can do here to solve the problem. Sorry
